src/main/java/org/broadinstitute/hellbender/tools/examples/ExampleVariantWalker.java

```diff
@@ -36,10 +37,17 @@
     public void onTraversalStart() {
         try {
             outputStream = outputFile != null ? new PrintStream(outputFile) : System.out;
-        }
-        catch ( final FileNotFoundException e ) {
+        } catch ( final FileNotFoundException e ) {
             throw new UserException.CouldNotReadInputFile(outputFile, e);
         }
+        if (readArguments.getReadInputs().stream().anyMatch(TaggedInputFileArgument::hasSymbolicName)) {
+            final Map<String, TaggedInputFileArgument> map = readArguments.getInputsBySymbolicName();
+            outputStream.println("reads inputs:");
+            for (final Map.Entry<String, TaggedInputFileArgument> kv : map.entrySet()){
+                outputStream.println("  " + kv.getKey() + ":" + kv.getValue().getFile().getName());
+            }
+            outputStream.println();
```

Comment:
The `ExampleReadWalkerWithVariants` version of this seems sufficient -- this seems out-of-place in `ExampleVariantWalker`